assets/cfg/app_config.json.dist

```diff
@@ -10,6 +10,17 @@
     "urlInstagram": "",
     "urlDonate": "",
     "keyMapTiler": "",
+    "mapDefaultZoom": 12.0,
+    "mapOfflineMinZoom": 8.0,
+    "mapOfflineMaxZoom": 14.0,
+    "mapOfflineZoom": 13.0,
+    "mapOnlineMinZoom": 1.0,
+    "mapOnlineMaxZoom": 19.0,
+    "mapOnlineZoom": 13.0,
+    "mapChooseLocationZoom": 16.0,
+    "mapCenterCoords": [-17.39000, -66.15400],
```

Comment:
I think we should make clear here, what's lat and what's lon ...

Reply:
Good point